# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rpds/hotfix-license.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rpds-0.23.1/pyproject.toml.vanilla	2025-03-21 18:43:54.846169001 +0100
+++ rpds-0.23.1/pyproject.toml	2025-03-21 18:43:58.482042204 +0100
@@ -8,7 +8,7 @@
 requires-python = ">=3.9"
 readme = "README.rst"
 license = "MIT"
-license-files = ["LICENSE"]
+license-files = {paths = ["/LICENSE"]}
 keywords = ["data structures", "rust", "persistent"]
 authors = [
   { name = "Julian Berman", email = "Julian+rpds@GrayVines.com" },
